test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+ansible==2.5.3.0
+molecule==2.19.0
+docker-py==1.10.6
```

Comment:
yesterday read that the Ansible recommendation is installing `docker` pip package, leaving `docker-py` for Python 2.6: https://docs.ansible.com/ansible/latest/scenario_guides/guide_docker.html#requirements

So I guess like 90% of our roles (or more) are just passing on this recommendation :laughing: